### PR TITLE
Expose Header On API

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

File: app/api/api/entities/organization.rb [hunk 1]
```diff
@@ -25,6 +25,9 @@ class Organization < Base
         expose :logo do |organization|
           organization.logo.attached? ? Rails.application.routes.url_helpers.url_for(organization.logo) : nil
         end
+        expose :image do |organization|
```
Comment (garyhtou) [inline app/api/api/entities/organization.rb]: can we call this `donation_header_image` or `donation_header`?

`image` is just a tad bit too broad

**Review by garyhtou — COMMENTED** [review]
looks great! just a small change

File: app/api/api/entities/organization.rb (hunk 1 above)
Comment (sampoder) [reply app/api/api/entities/organization.rb]: let's do it!

Comment (sampoder) [inline app/api/api/entities/organization.rb]:
```suggestion
        expose :donation_header do |organization|
```

**Comment by sampoder** [comment]
Got Ian and Ben's approval IRL.

**Pull request merged**